src/librustc_mir/util/pretty.rs

```diff
@@ -780,7 +780,7 @@ pub fn write_allocation<Tag, Extra>(
             i += Size::from_bytes(1);
         } else {
             write!(w, "__")?;
-            ascii.push(b'_');
+            ascii.push(b'.');
```

Comment:
We could use something cute from unicode if we want to, FWIW.

Reply:
yea, I considered that, but then I'd need to use a string here and not a `Vec<u8>` and need to convert here immediately to the final representation.

Reply:
I was looking for something like this but I don't know what it would be called:
```
\/\/\/
/\/\/\
\/\/\/
/\/\/\
```

Reply:
https://unicode-table.com/en/sets/arrow-symbols/

Maybe use a plain `↑` or a `↥`?

Reply:
I wonder if `░` or `▒` would work well enough.

Reply:
Oh, `▩` is what I was thinking of, but it's a bit too dense :disappointed:

Reply:
I used ░